### PR TITLE
Add Brazilian Portuguese

### PR DESCRIPTION
**Pull request by wowbaggerBR**
Fully translated .po file to pt_BR, .mo file compiled and I also updated the .desktop file with the necessary translations. Let me know if something needs to be fixed.

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -1,5 +1,208 @@
...
+msgid "Password:"
+msgstr "Senha:"
+
+#: ../src/Widgets/DataBaseOpen.vala:79
+msgid "Run Query"
+msgstr "Executar Query"
+
+#: ../src/Widgets/DataBaseOpen.vala:89
+msgid "Running Query..."
+msgstr "Executando Query"
```
Comment (rafaelgontijo) [inline po/pt_BR.po]: would it be better to add three dots?

File: po/pt_BR.po [hunk 2]
```diff
@@ -1,5 +1,208 @@
...
+msgid "No Results Available"
+msgstr "Nenhum resultado disponível"
+
+#: ../src/Widgets/DataBaseOpen.vala:144 ../src/Widgets/DataBaseOpen.vala:154
+msgid "Unable to process Query!"
+msgstr "Impossível executar a Query!"
+
+#: ../src/Widgets/DataBaseOpen.vala:146 ../src/Widgets/DataBaseOpen.vala:168
+msgid "Query Successfully Executed!"
+msgstr "Query Executada com Sucesso"
```
Comment (rafaelgontijo) [inline po/pt_BR.po]: would it be better to add point exclamation?

File: po/pt_BR.po [hunk 3]
```diff
@@ -1,5 +1,208 @@
...
+msgid "General"
+msgstr "Geral"
+
+#: ../src/Widgets/SettingsDialog.vala:51
+msgid "Interface"
+msgstr "Interface"
+
+#: ../src/Widgets/SettingsDialog.vala:80
+msgid "Automatically Save New Connections:"
+msgstr "Salvar Novas Conexões Automaticamente"
```
Comment (rafaelgontijo) [inline po/pt_BR.po]: would it be better to add two dots?

File: po/pt_BR.po [hunk 4]
```diff
@@ -1,5 +1,208 @@
...
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: pt_BR\n"
+
+#: ../src/Window.vala:123
+msgid "Welcome"
+msgstr "Bem-vindo"
+
+#: ../src/Window.vala:129
+msgid "Connection Saved!"
+msgstr "Conexão salva"
```
Comment (rafaelgontijo) [inline po/pt_BR.po]: would it be better to add point exclamation?

**Comment by wowbaggerBR** [comment]
You're absolutely right, thanks for pointing that out.

**Comment by rafaelgontijo** [comment]
It's great now.

**Comment by Alecaddd** [comment]
Thank you so much, guys!
I'm currently moving from CMake to meson build, which will allow me to automatically refresh the translation files because with the new features and the previous refactor, lots of copy is not part of the current language files.
Cheers

**Pull request merged**